Source/CCZ4/Weyl4.impl.hpp

```diff
@@ -259,7 +254,7 @@ Weyl4::compute_null_tetrad(const Vars<data_t> &vars,
     out.w[2] = 0.0;
 
     // floor on chi
-    const data_t chi = simd_max(vars.chi, 1e-4);
+    const amrex::Real chi = (vars.chi < 1e-4) ? 1e-4 : vars.chi;
```

Comment:
Maybe this should use the same same `min_chi` as used in `PositiveChiAndLapse`
but that's for another time. In any case:
```suggestion
    const amrex::Real chi = std::max(vars.chi, 1.0e-4_rt);
```

Reply:
Actually I will just remove this. It should be unnecessary as it is enforced in PositiveChiAndLapse as you say, and then there won't be any inconsistency.